src/components/Form/Form.tsx

```diff
@@ -1,28 +1,33 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Form.css';
 
 
 interface FormProps {
-  changeUserSentiment: (newSentiment: number | string) => void
+  changeUserSentiment: (newSentiment: number | null) => void
 }
 
 const Form = ({ changeUserSentiment } : FormProps) : JSX.Element => {
-  const assignSentiment = (event: React.MouseEvent<HTMLButtonElement>) => {
-      event.preventDefault();
-      const button: HTMLButtonElement = event.currentTarget;
-      changeUserSentiment(Number(button.name));
+  const assignSentiment = (sentiment:number) => {
+      changeUserSentiment(sentiment);
   }
 
   return(
     <section className="questionnaire">
-        <h2>how are you feeling today?</h2>
-        <div className="sentiment-selection">
-          <div className="sentiments-container">
-            <button className="happy-btn" name="1" onClick={assignSentiment}></button>
-            <button className="neutral-btn" name="0" onClick={assignSentiment}></button>
-            <button className="sad-btn" name="-1" onClick={assignSentiment}></button>
-          </div>
+      <h2>how are you feeling today?</h2>
+      <form className="sentiment-selection">
+        <div className="sentiments-container">
+          <Link to="/WellNewsFeed/FeelingGood" title="happy" onClick={() => assignSentiment(1)}>
```

Comment:
I checked out a few websites and it seems the convention for URLs is to make them lowercase and kebab-case (e.g., `/feed/feeling-good` instead of `/Feed/FeelingGood`